viam_cartographer.go

```diff
@@ -455,10 +455,6 @@ func (cartoSvc *CartographerService) GetPointCloudMap(ctx context.Context) (func
 		return nil, ErrClosed
 	}
 
-	if cartoSvc.SlamMode != cartofacade.LocalizingMode {
-		cartoSvc.mapTimestamp = time.Now().UTC()
```

Comment:
Does the comment above the function need to be updated (maybe remove "recording the time", and remove the "If start up is in localization mode...")?

Reply:
Done

Reply:
I don't think you have updated the comment

Reply:
Oops, done now